.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -1,13 +1,13 @@
 ---
 name: Bug
 about: Make sure you complete the template. Otherwise, it will be closed without further explanation!
-title: "[Bug]"
+title: "[Bug] (Replace this with your title)"
 labels: bug
 assignees: ''
 
 ---
 
-## ⚠️ Have you read the [FAQ](https://github.com/qnblackcat/uYouPlus/wiki/FAQ)? Make sure you visit the FAQ page first!!! ⚠️
+## ⚠️ Have you read the [FAQ](https://github.com/qnblackcat/uYouPlus/wiki/FAQ)? Make sure you visit the [FAQ](https://github.com/qnblackcat/uYouPlus/wiki/FAQ) page first!!! ⚠️
```

Comment:
Is it necessary to add the link to the second appearing term?

Reply:
I don't know, I thought that I might as well...